### DISCUSSION TITLE
Cannot export PDF with more than 400 pages

### DISCUSSION THREAD

**Discussion by a-asharapova** [GitbookIO/community#609] [5. Export]
Hello everyone!

Our colleagues cannot export the PDF generated for https://docs.parallels.com/parallels-ras-administrators-guide-19. According to GitBook, the PDF is 486 pages long, but the export stops at 400. This is a serious issue for us as this is a popular document. 

Please help us find a solution to this problem as soon as you can. Thank you!

**Comment by a-asharapova** [comment]
In addition, the fact that the page limit for PDF export can be increased only by 50 pages at a time makes the process frustrating for our colleagues and customers.

**Reply by aikoii** [reply]
Hey @a-asharapova, I just responded to your email 🙂, but for those experiencing PDF export issues, hard-refreshing their browser or clearing the cache might help as a temporary fix. I've also flagged this internally 🙏.

**Comment by a-asharapova** [comment]
Hello everyone!

Are there any updates on this issue?

**Reply by aikoii** [reply]
Hey @a-asharapova! No updates on this yet. Have you/your colleagues tried the above? Sorry this isn't ideal, but seems to help for now

**Reply by a-asharapova** [reply]
@aikoii Unfortunately, this method does not work for us at all. One person managed to load 421 pages, after which the button to load more pages disappeared. For others, it didn't work at all.

**Reply by aikoii** [reply]
@a-asharapova So sorry for missing this! Thanks for letting me know. Let me check this with the team 🙏

**Reply by aikoii** [reply]
@a-asharapova Not the best update, but exporting PDFs with more than 400 pages is a current limitation of our system. Unfortunately, there aren't any workarounds at this time besides exporting the document in smaller chunks of pages instead of the whole space.

**Comment by milothiesen** [comment]
Boosting this. A lot of our users really like to print things out on paper. Our docs are over 600 pages long!

**Comment by jasonspar** [comment]
Yeah, we also need this capability. 400 pages is a severe limitation in this day and age.

**Comment by JaniceManwiller** [comment]
This is also a problem for our team. Some of our customers can't access the doc site and we would like to send them a full PDF.

And there doesn't seem to be a way to identify the smaller chunks to export. If you could at least select a group of pages to export it would help, so that you know what you already had and could stitch together the PDFs.

Right now it's either export the current page or try to export the whole thing and add 50 pages at a time. If you reach the limit (have the patience to keep clicking "next 50 pages"), how do you know where you left off?
